scripts/alerts/validate_alerts_format.py

```diff
@@ -0,0 +1,70 @@
+import sys
+import yaml
+import json
+from google.cloud import monitoring_v3
+
+def check_json_in_metadata(path, file_id, file_version):
+  metadata_path = "/".join(path.split("/")[:-1]) + "/metadata.yaml"
+  check_metadata_entries(metadata_path)
+  f = open(metadata_path)
+  data = yaml.safe_load(f)
+  templates_metadata = data.get("alert_policy_templates")
+  for template_metadata in templates_metadata:
+    if template_metadata.get("id") == file_id and template_metadata.get("version") == int(file_version[1]):
+      return
+  raise Exception("{} does not have an entry in {}".format(path, metadata_path))
+
+def check_metadata_entries(path):
+  f = open(path)
+  data = yaml.safe_load(f)
+  templates_metadata = data.get("alert_policy_templates")
+  if not templates_metadata:
+    raise Exception("alert_policy_templates not defined in {}".format(path))
+  required_fields = ["id", "version", "display_name", "description"]
+  for template_metadata in templates_metadata:
```

Comment:
Might be clearer to use sets here. Something like:

    required_fields = set(...)
    missing_fields = required_fields - templates_metadata.keys()
    if missing_fields:
      raise Exception(...)

Reply:
good point - fixed